pom.xml

```diff
@@ -32,6 +32,7 @@
         <batik.version>1.17</batik.version>
         <logback.version>1.3.14</logback.version>
         <mockito.version>3.12.4</mockito.version>
+        <upload-service.version>1.0.0</upload-service.version>
```

Comment:
When Baratha merges and creates the release for the updated upload service client this version should be updated